RxCocoa/Traits/SharedSequence/SharedSequence+Concurrency.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SharedSequence+Concurrency.swift
+//  RxCocoa
+//
+//  Created by Shai Mishali on 22/09/2021.
+//  Copyright © 2021 Krunoslav Zaher. All rights reserved.
+//
+
+#if swift(>=5.5) && canImport(_Concurrency) && !os(Linux)
+import Foundation
+
+// MARK: - Shared Sequence
+@available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
```

Comment:
Could these new APIs be made available on older OSes too, as Xcode 13.2 brought Concurrency support for them? ([release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-13_2-release-notes#Swift))
`@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)`

That would also require a change in the `#if` statement:
`#if compiler(>=5.5.2) && canImport(_Concurrency)`

Reply:
Yup:) it's the next commit on the list

Reply:
See https://github.com/ReactiveX/RxSwift/pull/2367/commits/e498cbeb1239d4cba87680a4ef55912a06db36ce